### DISCUSSION TITLE
Multiple AND/OR filters

### DISCUSSION THREAD

**Discussion by hansblaauw64** [Budibase/budibase#12647] [Help]
We are having serious issues with the filtering options. We want to be able to use a filter on one data provider and then use that result as input for the next data provider which also has a filter defined and is connected to a table block. Basically we are creating an AND filter where we can also leave values empty to show all results.

Match all wouldn't work in the first filter actually.

Any suggestions?

**Comment by melohagan** [comment]
Hey @hansblaauw64 

I'm not sure I fully understand what you are asking. You can use the data provider rows as a binding. Something like this?

![Screenshot 2023-12-20 at 14 52 56](https://github.com/Budibase/budibase/assets/101575380/ed16e47a-a2b3-491d-b0b4-6c85f357471d)

Filter on the inner Data provider:
![Screenshot 2023-12-20 at 14 53 28](https://github.com/Budibase/budibase/assets/101575380/d3d621aa-e671-4ee8-a108-6ece7ac2a13a)

---

If you are specifically looking to do a combination of AND / OR filters within one Data provider, this is currently not possible: https://github.com/Budibase/budibase/issues/3183
